packages/oc-template-react-compiler/lib/compileServer.js

```diff
@@ -50,30 +50,48 @@ module.exports = (options, callback) => {
       next =>
         fs.outputFile(higherOrderServerPath, higherOrderServerContent, next),
       next => compiler(config, next),
-      (data, next) => fs.ensureDir(publishPath, err => next(err, data)),
       (data, next) => {
         const basePath = path.join(serverPath, "../temp/build");
-        const memoryFs = new MemoryFS(data);
-        const compiledServer = memoryFs.readFileSync(
-          `${basePath}/${config.output.filename}`,
-          "UTF8"
-        );
-        fs.writeFile(
-          path.join(publishPath, publishFileName),
-          compiledServer,
+        const memory = new MemoryFS(data);
```

Comment:
This is pretty similar to what we do on the server compiler: https://github.com/opencomponents/base-templates/blob/master/packages/oc-server-compiler/index.js#L28 with some little differences (path and source with the highorderserver)